buildbot.mariadb.org/scripts/deb-install.sh

```diff
@@ -1,18 +1,29 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2154
 
-set -ex
+set -e
```

Comment:
document arg is url.

Reply:
Your are right but only in the case of a manual run.